src/validate.ts

```diff
@@ -101,6 +101,11 @@ export function detectOpenAPI(root: any): OasVersion {
     throw new Error(`Document must be JSON object, got ${typeof root}`);
   }
 
+  if (!(root.openapi || root.swagger)) {
+    process.stderr.write('❗️❗️❗️ This doesn’t look like an OpenAPI document.\n');
```

Comment:
Other code around throws, your code `process.exit(1)`. Make it consistent please

Reply:
which way you think is better? as for ux i'd say that we can use exit(1) cause otherwise it outputs stacktrace which is not very useful for this case

Reply:
I would vote for throw Error. We can handle this error separately later.

There may be other definitions to be processed or it may be run in VS Code extension where `process.exit` doesn't make sense

Reply:
fair enough, didn't think of vs code or lib-like usage